models/gpg_key.go

```diff
@@ -64,9 +64,14 @@ func (key *GPGKey) AfterLoad(session *xorm.Session) {
 }
 
 // ListGPGKeys returns a list of public keys belongs to given user.
-func ListGPGKeys(uid int64) ([]*GPGKey, error) {
-	keys := make([]*GPGKey, 0, 5)
-	return keys, x.Where("owner_id=? AND primary_key_id=''", uid).Find(&keys)
+func ListGPGKeys(uid int64, listOptions ListOptions) ([]*GPGKey, error) {
+	sess := x.Where("owner_id=? AND primary_key_id=''", uid)
+	if listOptions.Page != 0 {
+		sess = listOptions.setSessionPagination(sess)
+	}
+
+	var keys []*GPGKey
```

Comment:
```suggestion
	keys := make([]*GPGKey, 0, 8)
```

Reply:
8 is probably a bit much for this one. Generally we'd expect people to have really 1 or 2 - so we could likely get away with 2 as a default.

Reply:
suggestion changed